demo/opae-nlb-demo/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Fedora is a recommended distro for OPAE
 # https://opae.github.io/latest
-ARG FEDORA_RELEASE="32"
+ARG FEDORA_RELEASE="33"
```

Comment:
32 and 33 are EOL. Is it possible to get to something that's maintained?

Reply:
another option would be to go back to `libjson-c5` from Debian as it was before b3aaecb771879763d323c41a5abbf37e4f293b07

Reply:
Used Fedora 34. Fedora 35 has incompatible libc with Debian.

Reply:
Fedora 34 EOL is soon. Hopefully Ubuntu 22.04 will be available at that time.